dapps/marketplace/src/pages/listing/_PaymentOptions.js

```diff
@@ -89,7 +91,7 @@ const PaymentOptions = ({
     needsSwap = false,
     noEthOrDai = false
 
-  if (hasIdentity) {
+  if (hasIdentity && !isHistoricalListing(listing)) {
```

Comment:
Why not move this check out of this component? We could avoid rendering `PaymentOptions` if it is a historical component right?

Reply:
`PaymentOptions` has children that it needs to render. If we disable the rendering of payment options then components like `_BuySingleUnit` and `Buy` do not get rendered either: https://github.com/OriginProtocol/origin/blob/5aa1d52bf362f8221ab6258f88f3899c773c461f/dapps/marketplace/src/pages/listing/_BuySingleUnit.js#L62-L75

Reply:
Yeah, But we don't want users to make offer on historical listing, if I'm not mistaken.

From what I understand, we would have a "View Current Listing" instead of "Purchase" button right?

Reply:
The button that displays the "View Current Listing" is a Buy button component. This ensures us that a user is never able to make an offer on a historical listing since the buy button will always say "View current listing" instead of "Purchase". 

And that buy button is part of the children that get passed to the PaymentOption Component.

Reply:
The button that displays the "View Current Listing" is a Buy button component. This ensures us that a user is never able to make an offer on a historical listing since the buy button will always say "View current listing" instead of "Purchase".

And that buy button is part of the children that get passed to the PaymentOption Component.